tools/no-bad-latin.py

```diff
@@ -0,0 +1,156 @@
+import os
```

Comment:
If we've copied from The Turing Way, the appropriate place to credit (IMO) would be here. Something like:

```suggestion
#!/usr/bin/env python
# 
# Detect Latin abbreviations that can be difficult for screenreaders and non-native English speakers
#
# This script initially adopted from The Turing Way from in October 2020.
# https://github.com/alan-turing-institute/the-turing-way/blob/af98c94/tests/no-bad-latin.py

import os
```

Reply:
Thanks.

Added the DOI and extended that to the other python file and the github action